tests/lib/systems.sh

```diff
@@ -14,16 +14,57 @@ is_core18_system(){
     return 1
 }
 
+is_core20_system(){
+    if [[ "$SPREAD_SYSTEM" == ubuntu-core-20-* ]]; then
+        return 0
+    fi
+    return 1
+}
+
 is_classic_system(){
     if [[ "$SPREAD_SYSTEM" != ubuntu-core-* ]]; then
         return 0
     fi
     return 1
 }
 
+
 is_ubuntu_14_system(){
     if [[ "$SPREAD_SYSTEM" == ubuntu-14.04-* ]]; then
         return 0
     fi
     return 1
 }
+
+get_snap_for_system(){
+    local snap=$1
+    local system=${2:-$SPREAD_SYSTEM}
```

Comment:
I'm kind of -1 on the second argument unless there's a really good reason to have it.

Reply:
done

Reply:
done